frontend/destino-cordoba/src/app/components/registro/registro.component.css

```
@@ -0,0 +1,42 @@
+.btn-registro {
+  border-radius: 4px;
+  background-color: #4286f4;
+  color: #fff;
+  padding: 10px;
+  border: none;
+  border-radius: 5px;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  padding-top: 5rem;
+  align-items: center;
+  height: 110vh;
```

```suggestion
  height: 100vh;
```
La altura mejor en 100 para que no haga el scroll